src/grammar-runtime-3/parser-2.ts

```diff
@@ -2482,3 +2610,102 @@ export function skipExpectation (
         addState(nextState);
     }
 }
+
+/**
+ * Will modify the `states` array.
+ */
+export function eagerSkipExpectation (
```

Comment:
This lets us skip the `Comma` token in MySQL in a greedy fashion. This lets us return `(Expected Comma) (WhiteSpace)` instead of `(WhiteSpace) (Expected Comma)`.

Both results are technically correct. But humans would rather see the former reported (as a matter of style).